src/test/fixtures/AccessResponse.json

```diff
@@ -0,0 +1,18 @@
+{
+  "newToken": "test_new_token",
+  "token": "test_new_token",
+  "user": {
+    "email": "luv@izadoab.ke",
+    "userId": "$2a$08$6BfcfDqh18sI07ngolt7dO4KZNJUBjWK6/e4q1jirTvK7NMa8vfi.",
+    "mnemonic": "Bill Hunt Sally Reed Bobby Powell Willie Johnson Ethan Christensen Lawrence Washington",
```

Comment:
Great to see fixtures with format as-is in real situations. Great on that @JoanVicens !